run_kraken.py

```diff
@@ -352,7 +355,8 @@ def main(cfg):
 
             iteration += 1
             logging.info("")
-
+        # send telemetry
+        telemetry.send_telemetry(config["telemetry"], chaos_telemetry)
```

Comment:
Output data being sent to the telemetry in stdout like we discussed. Maybe we can also add an option to output it to a file to save as artifacts for the run in case of CI. Thoughts?

Reply:
It should automatically go to cerberus.report too I believe